### PR TITLE
Updated Readme.md mentioning the packages to install for building Cheri

### PR DESCRIPTION
**Pull request by bitvijays**
Hello @arichardson,

Hope you are doing well. Have just updated the Readme.md for mentioning the packages to be installed for building Cheri on Ubuntu/ Debian box.

Let me know if any changes are required.

Kind Regards,
Vijay

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ It has been tested and should work on FreeBSD 10, 11 and 12.
 On Linux Ubuntu 16.04 and OpenSUSE Tubleweed are supported. Ubuntu 14.04 may also work but is no longer tested.
 MacOS 10.13 is also supported.
 
+# Pre-Build Setup
+
+If you are building Cheri on a Debian/ Ubuntu based machine, please install the below packages:
+
+```shell
+apt-get install libtool pkg-config clang bison libgtk2.0-dev cmake ninja-build samba flex
```
Comment (arichardson) [inline README.md]: I think `libgtk2.0-dev` is not needed, `libglib2.0-dev libpixman-1-dev` should be sufficient.

Comment (arichardson) [reply README.md]: Building CheriBSD will also require `libarchive-dev`,`bsdtar` and `libbz2-dev`

Comment (bitvijays) [reply README.md]: @arichardson Updated

**Comment by arichardson** [comment]
Thank you!

**Pull request merged**